web/src/main/kotlin/no/nav/su/se/bakover/web/routes/vilkår/utenlandsopphold/LeggTilUtenlandsoppholdRoute.kt

```diff
@@ -0,0 +1,182 @@
+import no.nav.su.se.bakover.web.features.authorize
+import no.nav.su.se.bakover.web.routes.Feilresponser
+import no.nav.su.se.bakover.web.routes.periode.toPeriodeOrResultat
+import no.nav.su.se.bakover.web.routes.revurdering.revurderingPath
+import no.nav.su.se.bakover.web.routes.revurdering.toJson
+import no.nav.su.se.bakover.web.routes.søknadsbehandling.behandlingPath
+import no.nav.su.se.bakover.web.routes.søknadsbehandling.toJson
+import no.nav.su.se.bakover.web.svar
+import no.nav.su.se.bakover.web.withBehandlingId
+import no.nav.su.se.bakover.web.withBody
+import no.nav.su.se.bakover.web.withRevurderingId
+import java.util.UUID
+
+internal fun Route.leggTilUtenlandsopphold(
+    søknadsbehandlingService: SøknadsbehandlingService,
+    satsFactory: SatsFactory,
+) {
+    post("$behandlingPath/{behandlingId}/utenlandsopphold") {
+        authorize(Brukerrolle.Saksbehandler) {
+            call.withBehandlingId { behandlingId ->
+                call.withBody<UtenlandsoppholdBody> { body ->
+                    body.tilLeggTilUtenlandsoppholdrequest(behandlingId).mapLeft {
+                        call.svar(it)
+                    }.map { request ->
+                        søknadsbehandlingService.leggTilUtenlandsopphold(request).fold(
+                            ifLeft = {
+                                call.svar(it.tilResultat())
+                            },
+                            ifRight = {
+                                call.svar(Resultat.json(HttpStatusCode.OK, serialize(it.toJson(satsFactory))))
+                            },
+                        )
+                    }
+                }
+            }
+        }
+    }
+}
+
+internal fun Route.leggTilUtlandsoppholdRoute(
+    revurderingService: RevurderingService,
+    satsFactory: SatsFactory,
+) {
+    post("$revurderingPath/{revurderingId}/utenlandsopphold") {
+        authorize(Brukerrolle.Saksbehandler) {
+            call.withRevurderingId { revurderingId ->
+                call.withBody<UtenlandsoppholdBody> { body ->
+                    body.tilLeggTilFlereUtenlandsoppholdRequest(revurderingId).mapLeft {
+                        call.svar(it)
+                    }.map {
+                        revurderingService.leggTilUtenlandsopphold(it)
+                            .fold(
+                                ifLeft = {
+                                    call.svar(it.tilResultat())
+                                },
+                                ifRight = {
+                                    call.svar(Resultat.json(HttpStatusCode.OK, serialize(it.toJson(satsFactory))))
+                                },
+                            )
+                    }
+                }
+            }
+        }
+    }
+}
+
+internal data class UtenlandsoppholdBody(
+    val vurderinger: List<UtenlandsoppholdVurderingBody>,
+) {
+    fun tilLeggTilUtenlandsoppholdrequest(behandlingId: UUID): Either<Resultat, LeggTilUtenlandsoppholdRequest> =
```

Comment:
Kanskje heller bare mappe lista likt for begge og heller gjøre den antall-sjekken i route-funksjonen?